gravatar/src/main/java/com/gravatar/restapi/models/AssociatedEmail200Response.kt

```diff
@@ -0,0 +1,45 @@
+/**
+ *
+ * Please note:
+ * This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * Do not edit this file manually.
+ *
+ */
+package com.gravatar.restapi.models
+
+import com.google.gson.annotations.SerializedName
+import java.util.Objects
+
+/**
+ *
+ *
+ * @param associated Whether the email is associated with a Gravatar account.
+ */
+
+public class AssociatedEmail200Response private constructor(
```

Comment:
`AssociatedEmail200Response` The OpenApi generator auto-generated this class name because we haven't defined an object for the endpoint response. I've opened a PR on the backend to refactor this.

Reply:
Yeah, makes sense - I saw the same name for a class in iOS.